File: 50 Coding Challenge Part II/JavaScript/Challenge41.js
// challenge 41: Create a function to calculate the distance between two points defined
// by their x, y coordinates

function getDistance(x1, y1, x2, y2){
    let line1 = x2 - x1;
    let line2 = y2 - y1;

    return Math.sqrt(Math.pow(line1, 2) + Math.pow(line2, 2))
}

// Driver Method
console.log(getDistance(2, 4, 4, 8));

File: 50 Coding Challenge Part II/Python/Challenge41.py
# Challenge 41: Create a function to calculate the distance between two points defined
# by their x, y coordinates
import math


def getDistance(x1, x2, y1, y2):
    return math.sqrt(math.pow(x2 - x1, 2) + math.pow(y2 - y1, 2))


# Driver Method
if __name__ == "__main__":
    print(getDistance(100, 100, 400, 300))
